source/css/_common/components/header/github-banner.styl

```diff
@@ -6,22 +18,19 @@
   :hover .octo-arm {
     animation: octocat-wave 560ms ease-in-out;
   }
-  @keyframes octocat-wave {
-    0%, 100% {
-      transform: rotate(0);
-    }
-    20%, 60% {
-      transform: rotate(-25deg);
-    }
-    40%, 80% {
-      transform: rotate(10deg);
-    }
+  > svg {
+    fill: bg_color;
+    color: #fff;
+    position: absolute;
+    top: 0;
+    border: 0;
+    right: 0;
   }
   +tablet-mobile() {
     > svg {
       if (scheme == 'Pisces') || (scheme == 'Gemini') {
-        fill: #fff !important;
```

Comment:
Here. Please, have a look.

Reply:
U say u fix `hide()` with `!important`. GitHub banner nothing to do with this.

Reply:
My mistake, it's not just about `hide()`
And I have tested, it's fine in all resolution, doesn't brake foreground styles